Tests/ResearchKitOnFHIRTests/FHIRToResearchKitTests.swift

```diff
@@ -14,11 +14,15 @@ import XCTest
 
 final class FHIRToResearchKitTests: XCTestCase {
     func testCreateORKNavigableOrderedTask() throws {
+        throw XCTSkip("Failing with a runtime error, seems to point to the other text option.")
```

Comment:
@vishnuravi The error can be more easily be reproduced here, even without the need for UI test to run.

Reply:
This error is due to the `ORKTextChoiceOther` now seemingly having a requirement that the text which is shown to the user equal the value of the choice. In our multiple choice questions we embed a `ValueCoding` in the value, but this is not strictly necessary for the other option, because the result item can just contain the string that was entered in the other field.

Reply:
Nice; thank you for resolving this @vishnuravi; amazing! 🚀